src/ignitus-Shared/ignitus-DesignSystem/ignitus-Organisms/ignitus-ProfileCard/Components/ProfileCard.jsx

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable jsx-a11y/accessible-emoji */
+import React from 'react';
+import * as S from '../styles';
+import { TextIconContainer } from '../../../shared';
+
+export default function ProfileCard() {
+  return (
+    <S.ProfileDiv>
+      <S.ProfileContainer>
+        <S.ProfileTrack>
+          <S.BookmarkIconDiv>
+            Track Sophi&apos;s progress
+            <S.ProfileBookmarkIcon />
+          </S.BookmarkIconDiv>
+        </S.ProfileTrack>
+        <S.ContentDiv>
+          <S.ImageContainer>
+            <img
+              src="https://storage.googleapis.com/ignitus_assets/ig-avatars/Team_avatar-eugene.png"
+              alt="ProfileLogo"
+            />
+          </S.ImageContainer>
+          <S.DetailsContainer>
+            <S.ProfileHeading>Sophia Carter</S.ProfileHeading>
+            <S.ProfileSubHeading>Literature Student</S.ProfileSubHeading>
+            <TextIconContainer>
+              <S.LocationIcon />
+              Location
+            </TextIconContainer>
+            <DropDownButton />
+          </S.DetailsContainer>
+        </S.ContentDiv>
+        <S.SocialDiv>
+          <S.SocialIcons>
+            <i className="fa fa-linkedin-square" />
+            <i className="fa fa-facebook-f" />
+            <i className="fa fa-github" />
+            <i className="fa fa-google-plus" />
+          </S.SocialIcons>
+          <S.SocialContact>
+            <TextIconContainer>
+              <S.SendMessageIcon />
```

Comment:
Use this approach - https://github.com/Ignitus/Ignitus-client/blob/master/src/ignitus-AboutPage/ignitus-CoreTeam/Components/team.tsx#L33